common/test/acceptance/tests/lms/test_lms.py

```diff
@@ -290,9 +294,19 @@ def test_deferred_verification_enrollment(self):
         enrollment_mode = self.dashboard_page.get_enrollment_mode(self.course_info["display_name"])
         self.assertEqual(enrollment_mode, 'verified')
 
+        # Expect enrollment activated event
+        self.assertEqual(
```

Comment:
I could also see it with `UniqueCourseTest`, but since that class is responsible specifically for providing tests with a unique course ID, I think it would be more appropriate to include these as generic helper methods (e.g., `assertExactlyOneEventEmitted`, `assertNoEventEmitted`) in [`helpers.py`](https://github.com/edx/edx-platform/blob/b59566a382d24274a24cf784ba1456f5f2dd07a3/common/test/acceptance/tests/helpers.py). It might also be better to have a single, more abstract helper, `assertEventsEmitted`, which takes an event name, a user ID, and a number of events to expect.

A separate PR works, but pulling it out now would save you some repeated code in this PR.